### PR TITLE
Post-Training Quantization of MobileNet v2 TensorFlow Model

### PR DESCRIPTION
**Pull request by alexsu52**
### Changes

Introducing the Post-Training Quantization of MobileNet v2 TensorFlow model example

### Reason for changes

Moving the example from the experimental module

### Related tickets

97478

### Tests

TBD

### CODE REVIEW

File: examples/post_training_quantization/tensorflow/mobilenet_v2/main.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    if verbose:
+        print(*str(cmd_output).split('\\n')[-9:-1], sep='\n')
+    match = re.search(r"Throughput\: (.+?) FPS", str(cmd_output))
+    return float(match.group(1))
+
+
+def get_model_size(ir_path: str, m_type: str = 'Mb', 
+                   verbose: bool = True) -> float:
+    xml_size = os.path.getsize(ir_path)
+    bin_size = os.path.getsize(ir_path.replace('xml', 'bin'))
```
Comment (daniil-lyakhov) [inline examples/post_training_quantization/tensorflow/mobilenet_v2/main.py:62]: Just to be confident
```suggestion
    bin_size = os.path.getsize(os.path.splitext(ir_path)[0] + 'bin')
```

File: examples/post_training_quantization/tensorflow/mobilenet_v2/main.py [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+def run_benchmark(model_path: str, shape: Optional[List[int]] = None, 
+                  verbose: bool = True) -> float:
+    command = f'benchmark_app -m {model_path} -d CPU -api async -t 15'
+    if shape is not None:
+        command += f' -shape [{",".join(str(x) for x in shape)}]'
+    cmd_output = subprocess.check_output(command, shell=True)
+    if verbose:
+        print(*str(cmd_output).split('\\n')[-9:-1], sep='\n')
+    match = re.search(r"Throughput\: (.+?) FPS", str(cmd_output))
```
Comment (daniil-lyakhov) [inline examples/post_training_quantization/tensorflow/mobilenet_v2/main.py:55]: Could it be simplified somehow?

File: examples/post_training_quantization/tensorflow/mobilenet_v2/main.py [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+model = tf.keras.applications.MobileNetV2(weights=weights_path, 
+                                          classes=DATASET_CLASSES)
+
+###############################################################################
+# Quantize a Tensorflow model
+
+'''
+The transformation function transforms a data item into model input data.
+
+To validate the transform function use the following code:
```
Comment (daniil-lyakhov) [inline examples/post_training_quantization/tensorflow/mobilenet_v2/main.py:134]: IMHO this line is a bit confusing. For me something like `The transform function is utilized in the following way` would be clearer

File: examples/post_training_quantization/tensorflow/mobilenet_v2/main.py [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+'''
+The calibration dataset is a small, no label, representative dataset
+(~100-500 samples) that is used to estimate the range, i.e. (min, max) of all
+floating point activation tensors in the model, to initialize the quantization
+parameters.
+
+The easiest way to define a calibration dataset is to use a training or
+validation dataset and a transformation function to remove labels from the data
+item and prepare model input data. The quantize method uses a small subset
+(default: 300 samples) of the calibration data set.
```
Comment (daniil-lyakhov) [inline examples/post_training_quantization/tensorflow/mobilenet_v2/main.py:151]:
```suggestion
The easiest way to define a calibration dataset is to use a training or
validation dataset and a transformation function to remove labels from the data
item and prepare model input data. Calibration dataset could be any type
but should be an iterable object and should yields pairs (input data, label).
The quantize method uses a small subset
(default: 300 samples) of the calibration data set.
```

File: examples/post_training_quantization/tensorflow/mobilenet_v2/main.py (hunk 2 above)
Comment (alexsu52) [reply examples/post_training_quantization/tensorflow/mobilenet_v2/main.py:55]: Could you make a proposal?

File: examples/post_training_quantization/tensorflow/mobilenet_v2/main.py (hunk 1 above)
Comment (alexsu52) [reply examples/post_training_quantization/tensorflow/mobilenet_v2/main.py:62]: done

File: examples/post_training_quantization/tensorflow/mobilenet_v2/main.py (hunk 3 above)
Comment (alexsu52) [reply examples/post_training_quantization/tensorflow/mobilenet_v2/main.py:134]: Makes sense, let's get back to it when the main documentation is leaked and I'll revise the comments.

File: examples/post_training_quantization/tensorflow/mobilenet_v2/main.py (hunk 4 above)
Comment (alexsu52) [reply examples/post_training_quantization/tensorflow/mobilenet_v2/main.py]: It's looks like this addition is wrong. Let's discuss it offline.

**Review by daniil-lyakhov — APPROVED** [review]

**Pull request merged**